### ISSUE TITLE
Disable selection highlight of table Cells for a particular column

### ISSUE THREAD

**Issue by DrewDennison** [palantir/blueprint#948]
I have a column in a bp table (fancy react table) that each cell contains a link. When I click the `Link` inside the cell, the cell is selected and highlighted before the url changes. I would like to be able to say that that column is not selectable or the highlight is disabled. I'm happy to provide a .gif if that's helpful.

**Comment by adidahiya** [comment]
Disabling the selection style is relatively easy to do (maybe an `allowSelection: boolean` prop on `Cell` which defaults to `true` and sets a CSS class), but removing it from the selected regions for interactions like copying cell contents might be tricky.

**Comment by cmslewis** [comment]
FWIW, when I added the reorder handle to column headers, I introduced an `ignoredSelectors` prop on the underlying `DragSelectable` component that accepts an array of string selectors that should _not_ trigger selection when clicked/`mousedown`'d. Would that do the trick here? (Imagine providing your `Column` or the link itself with a CSS class that you provide to `ignoredSelectors`.)

**Comment by DrewDennison** [comment]
Yes I think that would work. Low priority for me though. I don’t currently have any links anymore in my table but probably will in the future. Feel free to close that helps you.
